DEVELOPER_GUIDE.md

```diff
@@ -184,12 +184,17 @@ OTA_SUPPORT=1
 IoTConnect provides a signed URL that's valid for 5 minutes by default.
 Increasing this value will ensure that the URL does not expire before the download completes.
 * Update the chunk buffer to "uint8_t chunk_buffer[CY_OTA_CHUNK_SIZE + 1024]" at *mtb_shared/ota-update/\<version>/source/cy_ota_internal.h*.
```

Comment:
Please check if the latest version doesn't already have this size updated. Cleanup and check the whole section again.

Reply:
Cleaned up in new README.md